tests/runtime/test_services.py

```diff
@@ -51,6 +52,16 @@ def _fake_run(cmd, *args, **kwargs):
     return _fake_run
 
 
+def _mock_subprocess_module(monkeypatch):
+    mock_subprocess = Mock()
+    monkeypatch.setattr(
+        _services,
+        "subprocess",
+        mock_subprocess,
+    )
+    return mock_subprocess
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Any reason this is a function and not a fixture?

Reply:
It's used from within `_mock_subprocess_for_ray()` which is not a fixture itself because it accepts parameters. 

I think there is a way to make this work all with pytest fixtures, but the approaches I've seen make the code much more convoluted, so I opted for simple functions.